apps/design/frontend/src/export_screen.tsx

```diff
@@ -108,20 +114,71 @@ export function ExportScreen(): JSX.Element | null {
     });
   }
 
-  if (!electionPackageQuery.isSuccess) {
+  if (
+    !(electionPackageQuery.isSuccess && getBallotsFinalizedAtQuery.isSuccess)
+  ) {
     return null;
   }
   const electionPackage = electionPackageQuery.data;
   const isElectionPackageExportInProgress =
     exportElectionPackageMutation.isLoading ||
     (electionPackage.task && !electionPackage.task.completedAt);
 
+  const ballotsFinalizedAt = getBallotsFinalizedAtQuery.data;
+
   return (
     <ElectionNavScreen electionId={electionId}>
       <MainHeader>
         <H1>Export</H1>
       </MainHeader>
       <MainContent>
+        <H2>Ballots</H2>
+        <Column style={{ gap: '1rem' }}>
+          <InputGroup label="Ballot Template">
+            <SearchSelect
+              ariaLabel="Ballot Template"
+              value={getElectionQuery.data?.ballotTemplateId}
+              options={Object.entries(ballotTemplateOptions).map(
+                ([value, label]) => ({ value, label })
+              )}
+              onChange={(value) => {
+                setBallotTemplateMutation.mutate({
+                  electionId,
+                  ballotTemplateId: value as BallotTemplateId,
+                });
+              }}
+            />
+          </InputGroup>
+
+          <div>
+            <FieldName>Proofing Status</FieldName>
+            {ballotsFinalizedAt ? (
+              <Column style={{ gap: '0.5rem', alignItems: 'flex-start' }}>
+                <div>
+                  Ballots finalized at:{' '}
+                  {format.localeShortDateAndTime(ballotsFinalizedAt)}
+                </div>
+                <Button
+                  onPress={() => {
+                    setBallotsFinalizedAtMutation.mutate({
+                      electionId,
+                      finalizedAt: null,
+                    });
+                  }}
+                  disabled={setBallotsFinalizedAtMutation.isLoading}
+                  variant="danger"
+                  icon="Delete"
+                >
+                  Unfinalize Ballots
+                </Button>
```

Comment:
I was debating whether we need a confirmation for this, but honestly feels fine without as it's easy enough to just go back and re-finalize if you ever accidentally un-finalize

Reply:
I had the same thought process and conclusion